lib/cli/src/commands/package/build.rs

```diff
@@ -53,7 +54,10 @@ impl PackageBuild {
             )
         };
         let pkg = webc::wasmer_package::Package::from_manifest(manifest_path)?;
-        let pkg_hash = PackageHash::from_sha256_bytes(pkg.webc_hash());
+        let data = pkg.serialize()?;
+        let hash = sha2::Sha256::digest(&data).into();
+
+        let pkg_hash = PackageHash::from_sha256_bytes(hash);
```

Comment:
Won't this differ from the embedded webc hash in the file (because it doesn't ignore the hash in the file)?

I see the hash is only returned, not used, but could this cause issues for other callers?
Like the new deployment flow?

Reply:
The hash returned by `v2::OwnedReader` and `v3::OwnedReader`, implementors of `AbstractWebc`, is just a `digest` call on the underlying buffer equivalent to `data` here. So they should be the same.